### ISSUE TITLE
Error: No named parameter with the name 'primary'.

### ISSUE THREAD

**Issue by TercyoStorck** [leoshusar/mat_month_picker_dialog#5]
Trying to build, but get this error:

```
../../../../../.pub-cache/hosted/pub.dev/mat_month_picker_dialog-1.1.1/lib/src/mat_month_picker_dialog.dart:300:9: Error: No named parameter with the name 'primary'.
        primary: isSelected
        ^^^^^^^
../../../../../flutter/packages/flutter/lib/src/material/text_button.dart:146:22: Context: Found this candidate, but the arguments don't match.
  static ButtonStyle styleFrom({
                     ^^^^^^^^^
../../../../../.pub-cache/hosted/pub.dev/mat_month_picker_dialog-1.1.1/lib/src/mat_month_picker_dialog.dart:332:9: Error: No named parameter with the name 'primary'.
        primary: isSelected
        ^^^^^^^
../../../../../flutter/packages/flutter/lib/src/material/text_button.dart:146:22: Context: Found this candidate, but the arguments don't match.
  static ButtonStyle styleFrom({
                     ^^^^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.
```

Flutter version informations:
```
Flutter 3.19.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision abb292a07e (3 days ago) • 2024-02-20 14:35:05 -0800
Engine • revision 04817c99c9
Tools • Dart 3.3.0 • DevTools 2.31.1
```

**Comment by MMelito98** [comment]
I got the same issue

**Comment by leoshusar** [comment]
Please upgrade to 1.1.2, fixed in #6.
